src/components/Panel/Panel.scss

```diff
@@ -39,153 +46,103 @@ $CommandButtonBlue: #2488D8;
   position: fixed;
   right: 0;
   top: 0;
-  display: none; //prevent the panel from briefly showing up when the page loads
+  display: none;
   z-index: $ms-zIndex-front;
   width: 100%;
 
-  .ms-CommandBar {
-    .ms-CommandBarItem {
-      user-select: none;
-
-      &:hover {
-        background-color: $CommandButtonLighterBlue;
-      }
-
-      &:active {
-        background-color: $CommandButtonLightBlue;
-
-        .ms-CommandBarItem-icon {
-          color: $CommandButtonDarkBlue;;
-        }
-
-        .ms-CommandBarItem-commandText {
-          color: $ms-color-black;
-        }
-      }
-    }
-
-    .ms-CommandBarItem:first-child {
-      background-color: $ms-color-themePrimary;
-      box-shadow: inset 0 1px 0 0 $CommandButtonBlue;
-
-      .ms-CommandBarItem-icon {
-        color: $ms-color-white;
-      }
-
-      .ms-CommandBarItem-commandText {
-        color: $ms-color-white;
-      }
-
-      .ms-CommandBarItem-linkWrapper {
-        padding-left: 12px;
-        padding-right: 12px;
-      }
-
-      &:hover {
-        background-color: $ms-color-themeDark;
-        box-shadow: none;
+  // Medium screens and up - (anchored right, fixed width)
+  @media (min-width: $ms-screen-md-min) {
+    border-left: 1px solid $ms-color-neutralLight;
+    border-right: 1px solid $ms-color-neutralLight;
+    pointer-events: auto;
+    width: $Panel-width-sm;
+    @include drop-shadow(-30px, 0px, 30px, -30px, .2);
+    left: auto;
+  }
 
-        .ms-CommandBarItem-icon {
-          color: $ms-color-white;
-        }
+  .ms-CommandBar {
+    outline: 1px solid transparent;
 
-        .ms-CommandBarItem-commandText {
-          color: $ms-color-white;
-        }
-      }
+    // The commandbar is not displayed by default on md+ screens
+    @media (min-width: $ms-screen-md-min) {
+      display: none;
     }
   }
-}
 
-.ms-Panel-closeButton {
-  @include button-reset();
-  position: absolute;
-  right: 12px;
-  top: 12px;
-  height: 24px;
-  width: 24px;
-  outline: 0;
-  padding: 0;
-  color: $ms-color-neutralSecondary;
+  .ms-CommandBarItem {
+    margin-left: 8px;
 
-  &:hover {
-    color: $ms-color-neutralPrimary;
+    .ms-CommandBarItem-commandText {
+      display: inline-block;
+    }
   }
-}
 
-.ms-Panel-closeIcon {
-  font-size: $ms-font-size-m;
-}
-
-
-// Scrollable content area, below the command buttons.
-.ms-Panel-contentInner {
-  position: absolute;
-  top: 40px; // Push it below the commands.
-  bottom: 0;
-  left: 0;
-  right: 0;
-  padding: 0 32px 20px 32px;
-  overflow-y: auto;
-}
-
-// Header text at the top of the panel.
-.ms-Panel-headerText {
```

Comment:
What can we do about panels with long header text? See issue #320 

If we're limiting it to one line, we should probably truncate with ellipses at the end.